projects/ama-sdk/src/lib/helpers/utils/create-entries-names.ts

```diff
@@ -34,6 +34,11 @@ export const createProcessName = (name) => {
     return sanitizeString(name.replace(PROCESS_FILE_FORMAT, ''));
 };
 
+export const createRuleName = (name) => {
```

Comment:
Please use DecisionTable instead of Rule. Despite being synonyms, let's be consequent, either we call them DecisionTable or Rule